### ISSUE TITLE
Path to f8a_worker modules is not setup correctly for unit tests on CI

### ISSUE THREAD

**Issue by tisnik** [fabric8-analytics/fabric8-gemini-server#13]
f8a_worker repository seems to be installed correctly via `pip3 install` via the Dockerfile, but `pytest` can't find sources. Here are very simple reproducer of this problem:

```
"""Test module for classes and functions found in the utils module."""
import requests
import os

import src.utils

def test_validate_request_data():
    """Test the function validate_request_data."""
    pass
```

Excerpt from the log:

```
platform linux -- Python 3.4.5, pytest-3.4.1, py-1.5.2, pluggy-0.6.0 -- /root/payload/venv/bin/python3
cachedir: .pytest_cache
rootdir: /root/payload, inifile:
plugins: cov-2.5.1

collecting 0 items                                                             
collecting 1 item                                                              
collecting 1 item / 1 errors                                                   
collected 1 item / 1 errors                                                    


----------- coverage: platform linux, python 3.4.5-final-0 -----------
Name              Stmts   Miss  Cover   Missing
-----------------------------------------------
src/__init__.py       2      0   100%
src/rest_api.py      41     41     0%   2-75
src/utils.py        110    103     6%   9-224
-----------------------------------------------
TOTAL               153    144     6%

==================================== ERRORS ====================================
_____________________ ERROR collecting tests/test_utils.py _____________________
ImportError while importing test module '/root/payload/tests/test_utils.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
venv/lib/python3.4/site-packages/_pytest/python.py:411: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
venv/lib/python3.4/site-packages/py/_path/local.py:668: in pyimport
    __import__(modname)
venv/lib/python3.4/site-packages/_pytest/assertion/rewrite.py:213: in load_module
    py.builtin.exec_(co, mod.__dict__)
tests/test_utils.py:5: in <module>
    import src.utils
src/utils.py:8: in <module>
    from f8a_worker.models import OSIORegisteredRepos
E   ImportError: No module named 'f8a_worker'
!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!
=========================== 1 error in 0.62 seconds ============================
```

**Comment by GeetikaBatra** [comment]
@tisnik, It's not just CI, I am getting it in my local as well. I ran runtests.sh and got this 

```
Traceback (most recent call last):
  File "/Users/gbatra/fabric8-gemini-server/venv/lib/python3.6/site-packages/_pytest/config.py", line 329, in _getconftestmodules
    return self._path2confmods[path]
KeyError: local('/Users/gbatra/fabric8-gemini-server/tests')

During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/Users/gbatra/fabric8-gemini-server/venv/lib/python3.6/site-packages/_pytest/config.py", line 360, in _importconftest
    return self._conftestpath2mod[conftestpath]
KeyError: local('/Users/gbatra/fabric8-gemini-server/tests/conftest.py')

During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/Users/gbatra/fabric8-gemini-server/venv/lib/python3.6/site-packages/_pytest/config.py", line 366, in _importconftest
    mod = conftestpath.pyimport()
  File "/Users/gbatra/fabric8-gemini-server/venv/lib/python3.6/site-packages/py/_path/local.py", line 668, in pyimport
    __import__(modname)
  File "/Users/gbatra/fabric8-gemini-server/venv/lib/python3.6/site-packages/_pytest/assertion/rewrite.py", line 213, in load_module
    py.builtin.exec_(co, mod.__dict__)
  File "/Users/gbatra/fabric8-gemini-server/tests/conftest.py", line 8, in <module>
    from src.rest_api import *
  File "/Users/gbatra/fabric8-gemini-server/src/rest_api.py", line 5, in <module>
    from utils import DatabaseIngestion, scan_repo, validate_request_data
  File "/Users/gbatra/fabric8-gemini-server/src/utils.py", line 8, in <module>
    from f8a_worker.models import OSIORegisteredRepos
ModuleNotFoundError: No module named 'f8a_worker'
ERROR: could not load /Users/gbatra/fabric8-gemini-server/tests/conftest.py
```

**Comment by GeetikaBatra** [comment]
@tisnik. This will fix the issue. https://github.com/fabric8-analytics/fabric8-gemini-server/pull/14

**Comment by tisnik** [comment]
Fixed by #14 
Thanks a lot @GeetikaBatra
